be/.env.example

```diff
@@ -0,0 +1,8 @@
+Here is example .env file
+
+
+IP=[IP] database ip address
```

Comment:
you can fill out example with dev config. and .env comments are `#` symbols this prolly wont work at all

Reply:
https://github.com/vlucas/phpdotenv#comments

Reply:
Let's do this in a separate pr. I also have a lot of nitpicks with the env config but I reallly want this merged

Reply:
@Orzelius Agreed

Reply:
I'm resolving this and moving to #38